src/components/SQForm/SQFormReadOnlyField.tsx

```diff
@@ -23,13 +23,15 @@ function SQFormReadOnlyField({
     formikField: {field},
   } = useForm({name});
 
+  const getFieldValue = field.value || field.value === 0 ? field.value : '- -';
```

Comment:
I was kinda thinking it should just be checking for null or undefined...
`getFieldValue = field.value ?? '- -'`

Maybe I am missing something?

Reply:
@hvilander Agreed.  We would want to allow for things like false or the empty string.  I will update this.

Reply:
@hvilander Ok, have to go back to the drawing board a bit.  We can't do it that way because we NEED to render the '- -' when we have an empty string (which is falsy).  This is breaking our tests that are expecting to see a `- -` when we don't provide a value.  So we need to render the '- -' for `null`, `undefined`, and `''`.  Here is probably the nicest way to do it (although requires use of the loose equality operator which we may not want):

`field.value == null || field.value === '' ? '- -' : field.value`

This works since  `undefined == null`. Let me know if you can think of a better way to accomplish this

Reply:
What about 

`field.value !== null && field.value !== undefined && field.value.toString() ? field.value || '- -'`

Reply:
@20BBrown14 I like it, I think that should work.

Reply:
+1 to branden's solution. 
I am afraid of double equals and don't trust it.

Reply:
@aharpt throwing `!field.value && !Number.isInteger(field.value) ? '- -' : field.value` into the ring as well (i think it would work?), but this looks great and no change needed.